public/scripts/dashboard.js

```diff
@@ -23,23 +29,35 @@ function setDataIntoDashboard() {
             var status = doc.data().Status;
             var totalCompletedTask = doc.data().TotalCompletedTask;
 
-            displayDataIntoDashboard(totalDevelopmentTask, totalBusinessTask, totalMarketingTask, totalCompletedTask, startDate, endDate, status);
+            displayDataIntoDashboard(selectedDocument, totalDevelopmentTask, totalBusinessTask, totalMarketingTask, totalCompletedTask, startDate, endDate, status);
             getTasks();
         })
         .catch(function(error) {
             console.log("Error getting document:", error);
         });
 }
 
-function displayDataIntoDashboard(totalDevelopmentTask, totalBusinessTask, totalMarketingTask, totalCompletedTask, startDate, endDate, status) {
+function displayDataIntoDashboard(selectedDocument, totalDevelopmentTask, totalBusinessTask, totalMarketingTask, totalCompletedTask, startDate, endDate, status) {
 
-    document.getElementById("totalDevelopmentTask").innerHTML = totalDevelopmentTask;
-    document.getElementById("totalBusinessTask").innerHTML = totalBusinessTask;
-    document.getElementById("totalMarketingTask").innerHTML = totalMarketingTask;
-    document.getElementById("totalCompletedTask").innerHTML = totalCompletedTask;
-    document.getElementById("sprintStartDate").innerHTML = startDate;
-    document.getElementById("sprintEndDate").innerHTML = endDate;
-    document.getElementById("sprintStatus").innerHTML = status;
+    if (selectedSprint == backlogSprint) {
+        document.getElementById("sprintId").innerHTML = selectedDocument;
+        document.getElementById("totalDevelopmentTask").innerHTML = totalDevelopmentTask;
+        document.getElementById("totalBusinessTask").innerHTML = totalBusinessTask;
+        document.getElementById("totalMarketingTask").innerHTML = totalMarketingTask;
+        document.getElementById("totalCompletedTask").innerHTML = totalCompletedTask;
+        document.getElementById("sprintStartDate").innerHTML = "Empty";
+        document.getElementById("sprintEndDate").innerHTML = "Empty";
+        document.getElementById("sprintStatus").innerHTML = "Empty";
+    } else {
+        document.getElementById("sprintId").innerHTML = selectedDocument;
+        document.getElementById("totalDevelopmentTask").innerHTML = totalDevelopmentTask;
+        document.getElementById("totalBusinessTask").innerHTML = totalBusinessTask;
+        document.getElementById("totalMarketingTask").innerHTML = totalMarketingTask;
+        document.getElementById("totalCompletedTask").innerHTML = totalCompletedTask;
+        document.getElementById("sprintStartDate").innerHTML = startDate;
+        document.getElementById("sprintEndDate").innerHTML = endDate;
+        document.getElementById("sprintStatus").innerHTML = status;
+    }
```

Comment:
No if else needed here, Just like the data is coming for sprint from db, similarly for backlog it will come.
But backlog will "---" as empty string.

Reply:
then we have to put "---" in the database right?